ledger/accountdb.go

```diff
@@ -1343,58 +1443,63 @@ func (rd *resourcesData) GetAppParams() basics.AppParams {
 	}
 }
 
-func accountDataResources(
-	ctx context.Context,
-	accountData *basics.AccountData, rowid int64,
-	cb func(ctx context.Context, rowid int64, cidx basics.CreatableIndex, ctype basics.CreatableType, rd *resourcesData) error,
```

Comment:
I need this call-backed approach in catchpoint dump to reuse this function. so why removing?

Reply:
rolled back the change.